src/app/doc_exporter.cpp

```diff
@@ -1302,7 +1302,14 @@ void DocExporter::trimTexture(const Samples& samples,
         sample.isEmpty())
       continue;
 
-    bounds |= sample.inTextureBounds();
+    // We add the border padding in the sample size to do an union
+    // between full bounds and sample's inTextureBounds (it
+    // shouldn't make full bounds bigger).
+    gfx::Rect sampleBounds(sample.inTextureBounds());
+    sampleBounds.w += m_borderPadding;
+    sampleBounds.h += m_borderPadding;
```

Comment:
You can use:

```suggestion
    sampleBounds.inflate(m_borderPadding)
```

Or even avoid declaring a new variable:
```c++
bounds |= gfx::Rect(sample.inTextureBounds()).inflate(m_borderPadding);
```
Not sure if this is preferred or not...just mentioning the possibility.